.clang-format

```diff
@@ -57,7 +57,6 @@ ExperimentalAutoDetectBinPacking: false
 FixNamespaceComments: false
 ForEachMacros:
   - Q_FOREACH
```

Comment:
Do we use Q_FOREACH? That looks like a Qt thing. I assume we can get rid of it too?

Reply:
Yup, removed.

Reply:
Ah, nope. Turns out that removing this messes with formatting of our `foreach` functions:

```
static int foreach (lua_State* l)

UD::unbox(l, -2)->foreach ([l, ifunc](const K& k, const V& v)
```

These are methods where we want without a space (rather than the `for ()` space). It seems that having any value here is sufficient, but I think it's best to leave a useful value (`Q_FOREACH`) rather than random junk?

Reply:
Have gone with `FOREACH` rather than suggesting Qt support (Y)